install/share/copy-schema-to-ca.py

```diff
@@ -104,7 +104,7 @@ def restart_pki_ds():
     """Restart the CA DS instance to pick up schema changes
     """
     root_logger.info('Restarting CA DS')
-    services.service('dirsrv').restart(SERVERID)
+    services.service('dirsrv', api).restart(SERVERID)
```

Comment:
This is really special code, according docs, this code must be copied from RHEL7 to RHEL6 and then executed on RHEL6. S this change will break RHEL6. IMO this code should be set to stone as is :)